src/components/admin/agency/logo-upload.js

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+
+export default class LogoUpload extends React.Component {
+    state = {
+      loading: false,
+      logo: undefined,
+    };
+  
+    componentWillReceiveProps(nextProps) {
```

Comment:
`componentWillReceiveProps` is already deprecated: https://reactjs.org/docs/react-component.html#unsafe_componentwillreceiveprops